chain/src/tests/load_code_with_snapshot.rs

```diff
@@ -113,7 +113,7 @@ fn test_load_code() {
     let tx_status = tx_pool.get_tx_status(tx.hash());
     assert_eq!(
         tx_status.unwrap().unwrap(),
-        (TxStatus::Pending, Some(11174))
+        (TxStatus::Pending, Some(11325))
```

Comment:
the lock script's `hash_type` is `Data`, why is the cycle changed?

Reply:
I recompiled the lib used in this testcase with new compilation parameters, you can view the modification in script/testdata/Makefile